apps/admin/traefik2/ptl-intsvc/secret-provider.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: secrets-store.csi.x-k8s.io/v1alpha1
+kind: SecretProviderClass
+metadata:
+  name: traefik-default-cert
+  namespace: admin
+spec:
+  parameters:
+    keyvaultName: acmedcdcftappsprod
```

Comment:
Hey, hopefully a quick one for you today but changing over to acmedtscftptlintsvc keyvault and creating the cert. It's already been done for acmedtscftsboxintsvc